job-status-service/src/main/resources/templates/job_queue_html_template.ftl

```diff
@@ -30,7 +30,7 @@
         <div class="portalheader">
             <div class="container">
                 <a class="btn" role="button" href="https://portal.aodn.org.au">
-                    <img src="${aodnLogoLocation}" alt="Portal logo"/>
+                    <img src="https://s3-ap-southeast-2.amazonaws.com/static.emii.org.au/images/logo/AODN_logo_fullText.png" alt="Portal logo"/>
```

Comment:
The clean URL for the static website is: https://static.emii.org.au/images/logo/AODN_logo_fullText.png

Reply:
Is that url as stable as the other one? Like it goes through cloudfront right, so there's an extra element that could fail?

Reply:
No Cloudfront, no. The only issue would be if the bucket or the DNS record were changed, but that is in the "someone mucked up the infrastructure" and we can't design around that. static.emii.org.au should be considered stable!